lemur/certificates/schemas.py

```diff
@@ -353,6 +355,20 @@ def handle_subject_details(self, data):
                 data.pop("organization", None)
                 data.pop("organizational_unit", None)
 
+    @post_dump
+    def handle_certificate(self, cert):
+        if cert['root_authority']:
+            authority = cert['root_authority']
+        else:
+            authority = cert['authority']
+        authority_id = authority['id']
+        plugin = plugins.get(authority['plugin']['slug'])
+        cert['body'], cert['csr'], cert['chain'] = plugin.wrap_certificate(
+                cert,
+                authority_id
+        )
+        del cert['root_authority']
```

Comment:
why is cert['root_authority'] deleted?

Reply:
Just because I've also added:
    root_authority = fields.Nested(AuthorityNestedOutputSchema)

Without that get https://xxxxxx/lemur/api/1/certificates/1 has this new key.